user_modules/shared_testcases/testCasesForRAI.lua

```diff
@@ -0,0 +1,233 @@
+local testCasesForRAI = {}
+
+require('atf.util')
+local events = require("events")
+
+--[[@Connecttest_onlywith_StartSDL_InitHMIOnReady: replace original connecttest with new one
+-- the uses only RunSDL and InitHMI
+--! @parameters:
+--! @FileName - new name of connecttest
+--]]
+function testCasesForRAI:Connecttest_onlywith_StartSDL_InitHMIOnReady(FileName)
```

Comment:
@aderiabin, I agree with you that this can lead to complicate defects, but for now, as this structure of ATF, such approach is used in many scripts.
As example you can see in policy, many scripts starts with:
commonPreconditions:Connecttest_without_ExitBySDLDisconnect_WithoutOpenConnectionRegisterApp("connecttest_connect_device.lua")

In this function, I just remove also InitHMIOnReady(). 
I think that until we have stable common preconditions of ATF not using the ones from connecttest.lua, this is the easiest way.
If I don't use it I should perform redundant actions for StopSDL->StartSDL-> InitHMI...... and this may cause other errors in not related CRQ.